presto-main/src/main/java/com/facebook/presto/sql/planner/plan/AggregationNode.java

```diff
@@ -167,6 +169,19 @@ public double getConfidence()
         return groupingSets;
     }
 
+    @JsonProperty("partitioningRequirement")
+    public Set<Symbol> getPartitioningRequirement()
```

Comment:
I'm not sure I like this being in the node itself. It's a concern for the optimizers that have to make partitioning decisions.